packages/core/src/tracing/instrumentation/messaging/rdkafka.js

```diff
@@ -65,21 +71,10 @@ function logWarningForKafkaHeaderFormat(headerFormat) {
   // Trace correlation would be broken for rdkafka senders with the header format 'binary'. If that format has been
   // configured explicitly, we log a warning and ignore the config value. The rdkafka instrumentation alwas acts as if
   // format 'string' had been configured.
-  if (headerFormat === 'binary') {
+  if (headerFormat === 'binary' || headerFormat === 'both') {
```

Comment:
issue: I do not understand this change. We do not ignore "both" in v3.

Reply:
Binary headers are not supported in rd-kafka instrumentation only "string", see previous debug log.

Reply:
Sorry I just noticed that you just combined the code to one condition 👍